app/services/export/report.rb

```diff
@@ -54,8 +54,9 @@ def rows
       row << delivery_partner_organisation_rows.fetch(activity.id, nil)
       row << change_state_rows.fetch(activity.id, nil)
       row << actuals_rows.fetch(activity.id, nil) if actuals_rows.any?
-      row << variance_rows.fetch(activity.id, nil) if actuals_rows.any? && forecast_rows.any?
-      row << forecast_rows.fetch(activity.id, nil) if forecast_rows.any?
+      # can't call forecast_rows.any? because something is a map -- even if row value in each is (), still creates it, gives it ID, attaches empty list.
```

Comment:
We've sort of got the same comment here twice (again on line 102-3), and it's quite well captured in the commit message. I'm not sure we need either comment here as I think `has_forecast_rows` on line 101 is probably descriptive enough and hides implementation details, but do you think this is something visitors to the codebase particularly need to be aware of?

Reply:
no i think you're right, looking at the file as a whole and examining the commit message i think we're safe to get rid of both